CHANGES.rst

```diff
@@ -2,10 +2,18 @@
 CHANGES
 =======
 
-3.7.2 (unreleased)
+4.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Python 3.4, 3.5, 3.6 and PyPy.
+
+- Fix the ``evolve`` view to use independent transactions instead of
+  committing or aborting the thread-local current transaction.
```

Comment:
This view was calling `transaction.commit()` from within itself. If the transactions are being used in the new explicit mode, this would lead to an error when the caller tried to *also* commit the (now missing) transaction. So now the view uses its own independent transaction to avoid this.